src/linker/BannedSymbols.txt

```diff
@@ -1,2 +1,4 @@
 T:Mono.Cecil.Cil.ILProcessor;Use LinkerILProcessor instead
 M:Mono.Cecil.TypeReference.Resolve();Use LinkContext.Resolve and LinkContext.TryResolve helpers instead
+P:Mono.Collections.Generic.Collection`1{Mono.Cecil.ParameterDefinition}.Item(System.Int32); use x
+P:Mono.Cecil.ParameterDefinitionCollection.Item(System.Int32); use x
```

Comment:
What's this BannedSymbols file?

Reply:
Ah, I was looking into using the BannedApiAnalyzer to prevent indexing `MethodReference.Parameters`, but I couldn't get it to work. These changes shouldn't be there.